src/battlegoals/battleGoals.ts

```diff
@@ -1,38 +1,131 @@
+  'Winded: Never have fewer than three cards in your hand. (2 Checks)\n' +
+  'Acrobatic: Lose a card to negate 5 or more damage.\n' +
+  'Contagious: While afflicted by a negative condition, apply any negative condition to a monster.\n' +
+  'Perforated: Lose 6 or more hit points in a single round. (2 Checks)\n' +
+  'Pincushion: Get attacked by three or more monsters in the same round.\n' +
+  'Retaliator: Don\'t make any attacks in the scenario until after you\'ve taken damage for the first time.\n' +
+  'Sleepless: Take damage during the same round you take a long rest.\n' +
+  'Untouchable: Take no damage. (2 Checks)\n' +
+  'Wasteful: Lose a card to negate 2 or less damage.\n' +
+  'Drowsy: Long rest while at your maximum hit point value.\n' +
+  'Sober: Use no potions during the scenario.\n' +
+  'Covetous: Never collect a money token from end-of-turn looting.\n' +
+  'Miser: Never exit a room while money tokens remain in it. (2 Checks)\n' +
+  'Mugger: Kill a monster and loot its money token on the same turn.\n' +
+  'Pickpocket: Perform a loot action while adjacent to at least two monsters.\n' +
+  'Prosperous: Collect three or more money tokens in the same turn.\n' +
+  'Scavenger: Collect more money tokens than any other player.\n' +
+  'Spotless: End the scenario in a room with no remaining money tokens or treasure overlay tiles.\n' +
+  'Ambusher: Open a door and end your move action adjacent to a monster in the revealed room.\n' +
+  'Bastion: Be adjacent to at least two monsters while standing on a door hex.\n' +
+  'Compulsive: Every time you take a move action, move in a straight line.\n' +
+  'Fearful: Never end your turn adjacent to a monster.\n' +
+  'Hurdler: On your turn, enter three hexes containing monsters, traps, obstacles, hazardous or difficult terrain.\n' +
+  'Leader: Never enter a room with an ally already in it.\n' +
+  'Paranoid: End each of your turns adjacent to a wall or obstacle.\n' +
+  'Recluse: Never end your turn adjacent to an ally.\n' +
+  'Restless: Move at least one hex on each of your turns.\n' +
+  'Shadow: End each of your turns adjacent to an ally.\n' +
+  'Sociable: Never leave a room with a character ally remaining in it.\n' +
+  'Stalwart: Never leave a hex adjacent to a monster. (2 Checks)\n' +
+  'Thorough: Never exit a room with monsters remaining in it.\n' +
+  'Trailblazer: Enter two or more hexes of difficult terrain, hazardous terrain, or traps in a single turn.';
+
+export const communityBattleGoals = communityBattleGoalsAsString.split('\n')
+  .map(it => it.trim())
+  .map(it => it.split(':'))
+  .map(it => {
+    let text = it[1].trim();
+    const twoChecks = text.includes(' (2 Checks)');
+    text = text.replace(' (2 Checks)', '').replace('\n', '');
+    return {
+      displayName: it[0],
+      text,
+      reward: twoChecks ? 2 : 1
+    }
+  }).map((it, index) => new BattleGoal(it.displayName, it.reward, it.text, index, "empty"));
```

Comment:
From reading this code, the official battle goals have globalCardId id 258 to 281, and the satire gaming battle goals have globalCardId 0 to 63?

Reply:
I'm open to use different ids. What are you proposing?
Maybe even move away from `number` based ids and move to a composite id like
{ origin:string, id:number} as a type of its own?

Reply:
Yeah I think that would be nice!